packages/core/src/connectors/add-connectors.ts

```diff
@@ -0,0 +1,72 @@
+import { exec } from 'child_process';
+import { existsSync } from 'fs';
+import { mkdir, rename, unlink } from 'fs/promises';
+import path from 'path';
+import { promisify } from 'util';
+
+import chalk from 'chalk';
+import got from 'got';
+import rimraf from 'rimraf';
+import tar from 'tar';
+
+import { npmPackResultGuard } from './types';
+
+const execPromise = promisify(exec);
+
+const fetchOfficialConnectorList = async () => {
+  // Will change to "logto-io/connectors" once the new repo is ready.
```

Comment:
this could be done by npm actually? (e.g. list all packages under an org) can figure it out later

Reply:
Yeah, we can create a new org later, let's say `@logto-connectors`